recipes/openvino/all/conandata.yml

```diff
@@ -1,4 +1,20 @@
 sources:
+  "2024.3.0":
+    "openvino":
+      url: "https://github.com/openvinotoolkit/openvino/archive/refs/tags/2024.3.0.tar.gz"
+      sha256: "53fccad05279d0975eca84ec75517a7c360be9b0f7bcd822da29a7949c12ce70"
+    "arm_compute":
+      url: "https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v24.06.tar.gz"
```

Comment:
Hi @culhatsker, there is a new release of `arm_compute` https://github.com/ARM-software/ComputeLibrary/releases/tag/v24.07

BTW, how are you determining the association between openVINO versions and the resto of the libraries in sources?
Where is the compatibility exposed?

Reply:
Hi, @perseoGI, thanks, thanks for noting that, it is true that there's a newer version, but I think it is reasonable to use the version fixed in the release as it is the version that was used to test the product.

I asked the team, they advance the version of arm compute library quite often, but at the time of code freeze it was still version 26.06.

Reply:
Okey, I understand. I'm approving this PR then :)